### PR TITLE
Update TWCERT/CC ids

### PR DESCRIPTION
**Pull request by twcertcc-cnaadmin**
Update IDs:
1. CVE-2019-11064
2. CVE-2019-13405
3. CVE-2019-13406
4. CVE-2019-13407
5. CVE-2019-13408
6. CVE-2019-11060
7. CVE-2019-11061
8. CVE-2019-11063

### CODE REVIEW

**Comment by ccoffin** [comment]
Per the CNA rules (http://cve.mitre.org/cve/cna/rules.html), CVE entries must contain the minimum data requirements (http://cve.mitre.org/cve/cna/rules.html#Appendix_B). To process request this pull request and populate the entry(s) for these CVE IDs to the CVE master list, please address the items identified below.

- [DESCRIPTION] should describe the vulnerability with sufficient detail as to demonstrate that the vulnerability is unique. The minimum required information should be included in the [DESCRIPTION], as well as other details that are relevant or necessary to show uniqueness.

The description field is what we use to populate the CVE entry on the CVE web site. In the JSON, this comes from the description container (e.g., $.description.description_data.value). Please update the description to include the affected product and versions in each case.

Online resources; 
- Minimum data requirements and approved formats are documented in the CNA Rules, http://cve.mitre.org/cve/cna/rules.html#Appendix_B.   
- Submitting CVE entry(s), https://cve.mitre.org/cve/cna.html#submitting_cve_entry_info

If you have any questions, please do not hesitate to contact us.

Regards,

Chris

File: 2019/13xxx/CVE-2019-13405.json [hunk 1]
```diff
@@ -0,0 +1,92 @@
+{
```
Comment (cve-team) [inline 2019/13xxx/CVE-2019-13405.json]: A language tag was used multiple times: problemtype value #0

Comment (yinnping) [reply 2019/13xxx/CVE-2019-13405.json]: I have been fixed this validation error.
Should I need to do other operations?

**Comment by cve-team** [comment]
Thank you for your submission. Expect CVE-2019-11064, CVE-2019-13405, CVE-2019-13406, CVE-2019-13407, CVE-2019-13408, CVE-2019-11060, CVE-2019-11061, and CVE-2019-11063 to be published on http://cve.mitre.org in the next few hours.

**Pull request merged**